platform/darwin/src/MGLStyleLayer.h

```diff
@@ -63,12 +63,12 @@ MGL_EXPORT
 @property (nonatomic, assign, getter=isVisible) BOOL visible;
 
 /**
- The maximum zoom level at which the layer gets parsed and appears.
+ The exclusive upper bound of the range of zoom levels within which a map would display the layer.
```

Comment:
Hrm, the meaning of this is still a little hard to parse — I’m sure it’s technically correct (which is the best kind of correct), but could we offer an example or make it somehow simpler?